script/generate_build_info.py

```diff
@@ -130,6 +131,28 @@ def get_maintainer_from_dockerfile(dir_name):
                     return maintainer
     return "Unknown"
 
+def get_default_build_script(build_scripts_versions):
+    #print(build_scripts_versions[-1]['version'])
+    print((build_scripts_versions))
```

Comment:
I think this statement is not giving any useful information, can this be converted to show correctly formatted information? If not i think it can be safely removed